### ISSUE TITLE
Backblaze B2 Bucket with CloudFlare CDN no egress fees

### ISSUE THREAD

**Issue by NicksMacLife** [owncast/owncast#1733]
If a testing and detailed documentation can be done for BackBlaze Buckets using CloudFlare as the CDN I believe that this would be of great help and interest to the community as Backblaze has some of the best pricing for S3 storage and using CloudFlare CDN/DNS you do not pay for egress, which makes the extremely cost effective if you're looking to stream on the cheap.

**Comment by funkyhippo** [comment]
Figured I'd leave a comment here for developers in the future, but this combination doesn't work well. Ultimately, it violates CF's ToS section 2.8, causing it to trigger a redirect on all your stream resources to CF's abuse page, despite disabling caching on CF's end and also adding proper cache control response headers. This occurred mid-stream to about 100 viewers.

It seems CF's ToS is intentionally vague about non-HTML content, which is a bit unfortunate since we had to find out by triggering the violation during a stream.
